src/components/CityDetails.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import Table from "react-bootstrap/Table";
+
+export default class CityDetails extends React.Component {
+  render() {
+    const { currentCity } = this.props;
+    const { display_name, lat, lon } = currentCity;
+    return (
+      <>
+        <Table borderless>
```

Comment:
I really like the way you designed the details component. Really clean to put the details in a table.